Week1/homkework/app.js

```diff
@@ -0,0 +1,76 @@
+const mysql = require('mysql');
+const executeQuery = require('./executeQuery');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'mysql1',
+});
+
+connection.connect();
+
+executeQuery.fnExecuteQuery(connection, 'CREATE DATABASE world');
+executeQuery.fnExecuteQuery(connection, 'use world');
+
+executeQuery.fnExecuteQuery(
+  connection,
+  'create table country(\
+    name VARCHAR(50),\
+    continent VARCHAR(25),\
+    region VARCHAR(50),\
+    surfacearea INT,\
+    indepyear INT,\
+    population INT,\
+    lifeexpectancy INT,\
+    gnp INT,\
+    gnpold INT,\
+    localname VARCHAR(50),\
+    govermentform VARCHAR(50),\
+    headofstate VARCHAR(50),\
+    capital VARCHAR(50)\
+    )',
+);
+
+executeQuery.fnExecuteQuery(
+  connection,
+  'create table city (id INT, name varchar(50), countrycode varchar(5), district varchar(25),population int )',
+);
+
+const country_values = [
+  "insert into  country values ( 'Netherlands','Europe','Western Europe',42508, 1945, 17000000,82,901,890,'Nederlands','parliamentary','Willem-Alexander','Amsterdam')",
+  "insert into country values ( 'Nepal','Asia','Southern asia',147181.00,1769,23930000,57.8,4768.00,4837.00,'Nepal','Constitutional Monarchy','Gyanendra Bir Bikram','Kathmandu')",
+  "insert into country values ( 'Uruguay','South America','South America',175016.00,1828,3337000,75.2,20831.00,19967.00,'Uruguay','Republic','Jorge Batlle','Montevideo')",
```

Comment:
Here you have inserted float (nepal 57.8 for life expectancy) into a int value, try to see what's happened in the table ?

Reply:
But the insert is good, only the create needs to be changed :)